src/_pytest/helpconfig.py

```diff
@@ -113,7 +112,7 @@ def pytest_cmdline_parse() -> Generator[None, Config, Config]:
             % (
                 pytest.__version__,
                 ".".join(map(str, sys.version_info)),
-                os.getcwd(),
```

Comment:
this needs a change in the debug str, i would recommend to also add cwd to the debug mix as its a good to know if cwd and invocation_dir are different

Reply:
Good idea, I'll keep `cwd` as is and add `invocation_dir`.